BrainPortal/app/views/background_activities/new.html.erb

```diff
@@ -0,0 +1,258 @@
+
+<%-
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2024
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+-%>
+
+
+<% title 'Schedule Maintenance Activity' %>
+
+<h2>Schedule Maintenance Activity</h2>
+
+<%= error_messages_for @bac, :object_name => "activity" %>
+
+<% @bac.options ||= {} %>
+<%= hidden_field_tag 'background_activity[options][dummy]','val' %>
+
+<%= form_for @bac, :as => :background_activity, :url => { :action => "create" }, :datatype => "script" do |f| -%>
+
+  <div class="generalbox">
+
+    <p><%= f.label :user_id, "User" %><br>
+    <%= user_select("background_activity[user_id]", :selector => @bac.user_id.to_s) %>
+
+    <p><%= f.label :remote_resource_id, "Portal or Execution Server" %><br>
+    <%= bourreau_select("background_activity[remote_resource_id]", {
+       :bourreaux => RemoteResource.all.order(:name).to_a,
+       :selector => @bac.remote_resource_id.to_s } ) %>
+
+    <p><%= f.label :start_date, "Initial Start Date" %><br>
+    <%= text_field_tag :start_date, @start_date, :class => "datepicker" %>
+
```

Comment:
Is it mandatory? Can you mark it somehow? Or, maybe default to current day. Time could be 5 min ahead of the current or so

Reply:
When scheduling, yes, start_at is mandatory

Reply:
The calendar picker will already default to today.